### PR TITLE
Added changes to override the `commandTimeout` caps to resolve issues with default time out of `5s`

### PR DESCRIPTION
**Pull request by bipin-k**
[*] Added changes to override the `commandTimeout` caps to resolve issues with default time out of `5s`

### CODE REVIEW

**Comment by bipin-k** [comment]
@jlipps This change couldn't be tested locally as the below issue is yet to be resolved.
https://github.com/headspinio/appium-altunity-plugin/issues/8

File: package.json [hunk 1]
```diff
@@ -5,7 +5,7 @@
     "appium",
     "unity"
   ],
-  "version": "1.3.2",
```
Comment (jlipps) [inline package.json]: in a PR you shouldn't do things like update the version, that is for the maintainer to do when something is published

File: src/client/index.ts [hunk 2]
```diff
@@ -127,7 +128,8 @@ export default class AltUnityClient {
         this.conn = new Connection({
             log: opts.log,
             port: opts.port,
-            host: opts.host
+            host: opts.host,
+            commandTimeout : opts.commandTimeout
```
Comment (jlipps) [inline src/client/index.ts]: please follow the code style for spacing

**Review by jlipps — CHANGES_REQUESTED** [review]
change itself looks good, couple small things to fix

**Review by jlipps — APPROVED** [review]

**Pull request merged**